text/0070-metric-bound-instrument.md

```diff
@@ -0,0 +1,63 @@
+# Rename metric instrument Handles to "Bound Instruments"
+
+*Status: proposed 11/26/2019*
+
+The OpenTelemetry metrics API specification refers to a concept known
+as ["metric handles"](0009-metric-handles.md), which is a metric
+instrument bound to a `LabelSet`.  This OTEP proposes to change that
+term to "bound instruments" to avoid the more-generic term "handle".
+
+The corresponding method to create a bound instrument will be renamed
+"Bind" as opposed to "GetHandle".
+
+## Motivation
+
+The term "Handle" is widely seen as too general for its purpose in the
+metrics API.  Rather than re-use a widely-used noun for this concept,
+we instead will re-use the metric "instrument" noun and apply an
+adjective, "bound" to convey that it has been bound to a `LabelSet`.
+
+## Explanation
+
+"Handle" has been confusing from the start. However it was preceded by
+other potentially confusing terms (e.g., "TimeSeries", "Entry").  The
+term "Bound instrument" was initially suggested
+[here](https://github.com/open-telemetry/opentelemetry-specification/pull/299#discussion_r334211154)
+and widely accepted.
+
+## Internal details
+
+This is a simple renaming.  All uses of "handle" will be replaced by
+"bound instrument" in the specification.  All uses of the `GetHandle`
+method become `Bind`.
```

Comment:
How about `GetBoundInstrument()` as it makes it more clear that this method returns something. (like `GetHandle()` .

`Bind()` may be wrongly interpreted as an action or something one can use to manually bind instrument and labelset as in like `meter.Bind(labelset, instrument)`

Reply:
It is an action, it creates a new object, bound instrument.

Reply:
It is an action, but @cijothomas is pointing out that "Bind" doesn't imply that it returns something.

I could be convinced to move the `Bind()` method to the `Meter` as you have done, and to use a `GetBoundX` method on the instrument, but this method will just call `Bind()` (which will return something) and I'm not sure it's an improvement in clarity.